README.md

```diff
@@ -225,6 +225,21 @@ ttyplot also supports *counter* style metrics, calculating *rate* by measured ti
 &nbsp;
 
 
+## key bindings
+
+
+when reading data from a pipe, ttyplot accepts the following commands typed at the terminal:
+
+```
+q   quit
+r   toggle "rate mode" on and off
+```
+
+these commands do not work if the standard input is a terminal: in this case quit with Ctrl-C.
```

Comment:
If this section is kept after syncing help output, please use `<kbd>...</kbd>` wraping for Ctrl and for C, each.

Reply:
OK. Done.

Reply:
@edgar-bonet thanks! :+1: